lavalink/websocket.py

```diff
@@ -67,8 +67,8 @@ def connected(self):
                 await self._node._manager._node_connect(self._node)
                 asyncio.ensure_future(self._listen())
 
-                if not self._resuming_configured and self._resume_key is not None \
-                        and self._resume_timeout:
+                if not self._resuming_configured and self._resume_key \
+                        and (self._resume_timeout is not None and self._resume_timeout > 0):
```

Comment:
Soooo... basically the same thing than `if self._resume_timeout` ? 

(expected for negative values)

Reply:
None checks exists in case users try to be funny and pass `None`. This means I can safely do a greater-than check on the provided value.